src/runtime_src/tools/scripts/xrtdeps-win22.py

```diff
@@ -67,6 +67,7 @@ def main():
   parser.add_argument('--icd', action="store_true", help='install Khronos OpenCL icd loader library')
   parser.add_argument('--opencl', action="store_true", help='install Khronos OpenCL header files')
   parser.add_argument('--gtest', action="store_true", help='install gtest libraries')
+  parser.add_argument('--python', action="store_true", help='install python dependencies (pybind11)')
```

Comment:
The option "--python" suggests that python will be installed as opposed to its dependencies.  

Is there a reason why "--pybind11" wasn't used?

Reply:
My initial thought was it's a bit of future proofing in case we need more python packages in the future, but that's a good point -- changed it to pybind11